phoenix_liveview/phoenix_liveview.exs

```diff
@@ -0,0 +1,134 @@
+Application.put_env(:phoenix, :json_library, Jason)
+Application.put_env(:esbuild, :version, "0.12.18")
+
+Mix.install(
+  deps = [
+    {:phoenix, "~> 1.6.0-rc.0", override: true},
+    :jason,
+    :plug_cowboy,
+    {:phoenix_live_view, "~> 0.16.0"},
+    {:esbuild, "~> 0.2"}
+  ]
+)
+
+Code.require_file("mix_install_deps_backport.exs")
+
+Application.put_env(:esbuild, :default,
+  args: ~w(app.js --bundle --target=es2016 --outdir=../priv/static/assets),
+  cd: Path.expand("assets", __DIR__),
+  env: %{"NODE_PATH" => InstallFolderTemporaryBackport.determine_build_folder(deps) <> "/deps"}
+)
+
+Application.put_env(:live_clock, LiveClock.Endpoint,
+  server: true,
+  http: [ip: {127, 0, 0, 1}, port: 8081],
+  secret_key_base: "vuLgz/lXn+03HJIPTHbTMeZGd16UzvFxLgThphnLdafNmlZqCSGEZJe3Hp9cRhVs",
+  live_view: [signing_salt: "xF2dLhep"],
+  debug_errors: true,
+  watchers: [
+    esbuild: {Esbuild, :install_and_run, [:default, ~w(--sourcemap=inline --watch)]}
+  ]
+)
+
+defmodule LiveClock.ClockLive do
+  use Phoenix.LiveView
+  use Phoenix.HTML
+  import Phoenix.LiveView.Helpers
+
+  def render(assigns) do
+    ~L"""
```

Comment:
I had to use `~L` instead of `~H` because I could not get `~H` to work properly (it complained about unclosed attribute).

Reply:
fyi: https://github.com/phoenixframework/phoenix/issues/4449#issue-982872797